tests/utils/v2Data/request.js

```diff
@@ -0,0 +1,77 @@
+const { ResponseContainer, RequestContext } = require('../../../libV2/models');
+
+class ExpressResponseStub {
+    constructor() {
+        this._status = null;
+        this._body = null;
+        this._redirect = null;
+    }
+
+    status(code) {
+        this._status = code;
+        return this;
+    }
+
+    sendStatus(code) {
```

Comment:
Is there a need for `sendStatus()` when `status()` does the same?

Reply:
This class is meant to mock the Express response object. In that context `status` sets the status code and requires a call to `send` to actually send the response, while `sendStatus` will set the code and send an empty body for you.